//package com.sincetimes.redwar.game.comnunicate.protocol;
message RseQuerySendWishItem
{
	optional int32 ret = 1;
}